### PR TITLE
Add support for game providers for Games

### PR DESCRIPTION
**Pull request by strycore**
Contains tasks to match MAME game and auto create Lutris games from MAME

### CODE REVIEW

File: games/views/games.py [hunk 1]
```diff
@@ -65,7 +63,7 @@ def get_serializer_class(self):
             return serializers.GameInstallersSerializer
         return serializers.GameSerializer
 
-    def get(self, request):
+    def get(self, _request):
```
Comment (AlexanderRavenheart) [inline games/views/games.py:66]: Why did you add _ to request?

File: games/views/games.py [hunk 2]
```diff
@@ -124,7 +124,8 @@ class GameStatsView(APIView):
     """View for game statistics"""
     permission_classes = (permissions.IsAdminUser, )
 
-    def get(self, request, format=None):
+    @staticmethod
+    def get(_request, _format=None):
```
Comment (AlexanderRavenheart) [inline games/views/games.py:128]: Why did you make get a staticmethod and added _ to the parameters?

File: games/views/games.py (hunk 1 above)
Comment (strycore) [reply games/views/games.py:66]: to remove a pylint warning

File: games/views/games.py (hunk 2 above)
Comment (strycore) [reply games/views/games.py:128]: gets rid of pylint warnings
